IntelliJ/Nx/build.gradle.kts

```diff
@@ -0,0 +1,176 @@
+plugins {
+    id("java")
+}
+
+group = "com.nuix.nx"
+version = "1.19.0-SNAPSHOT"
+
+val sourceCompatibility = 11
+val targetCompatibility = 11
+
+// Directory containing Nuix Engine release
+val nuixEngineDirectory: String = System.getenv("NUIX_ENGINE_DIR")
+println("NUIX_ENGINE_DIR: ${nuixEngineDirectory}")
+if (nuixEngineDirectory.isEmpty()) {
+    throw InvalidUserDataException("Please populate the environment variable 'NUIX_ENGINE_DIR' with directory containing a Nuix Engine release")
+}
+
+val engineLibDir = "${nuixEngineDirectory}\\lib"
+println("engineLibDir: ${engineLibDir}")
+
+val nuixAppLibDir = "C:\\Program Files\\Nuix\\Nuix 100.2\\lib"
+println("nuixAppLibDir: ${nuixAppLibDir}")
+
+repositories {
+    mavenCentral()
+}
+
+// We can use this to define JAR files that we need copied to lib
+val externalDependency: Configuration by configurations.creating {
+    isTransitive = true
+}
+
+dependencies {
+    implementation("org.jetbrains:annotations:24.0.1")
+
+    compileOnly("org.projectlombok:lombok:1.18.26")
+    annotationProcessor("org.projectlombok:lombok:1.18.26")
+
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.9.2")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.9.2")
+
+    // ====================
+    // Engine Dependencies
+    // ====================
+
+    implementation(fileTree(baseDir = engineLibDir) {
+        include(
+                "**/*log*.jar",
+                "**/*aspect*.jar",
+                "**/*joda*.jar",
+                "**/*commons*.jar",
+                "**/*guava*.jar",
+                "**/*gson*.jar",
+                "**/nuix-*.jar",
+                "**/*jruby*.jar",
+                "**/*swing*.jar",
+                "**/*jide*.jar",
+                "**/*csv*.jar",
+                "**/*beansbinding*.jar",
+                "**/*xml.bind*.jar",
+                "**/*itext*.jar",
+        )
+    })
+
+    testRuntimeOnly(fileTree(baseDir = engineLibDir) {
+        include("*.jar")
+    })
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
+    }
+}
+
+fun configureTestEnvironment(test: Test) {
+    println("Running 'configureTestEnvironment'...")
+
+    // Engine runtime temp directory
+    val nuixTempDirectory = findProperty("tempDir") ?: "${System.getenv("LOCALAPPDATA")}\\Temp\\Nuix"
+
+    // Args passed to JVM running tests
+    test.jvmArgs(
+            "--add-exports=java.base/jdk.internal.loader=ALL-UNNAMED",  // Engine 9.6(?) and later require this
+            "-Xmx4G",
+            "-Djava.io.tmpdir=\"${nuixTempDirectory}\"",
+            // "-verbose:class" // Can help troubleshoot weird dependency issues
+    )
+
+    // Directory used to store data a test may rely on (like sample data)
+    val testDataDirectory = "${projectDir}\\..\\..\\TestData"
```

Comment:
Can you change the back slashes ("\\") to forward slashes ("/").  "/" will work on Windows, but "\\" will not work on *nix.  Or consider using a path builder like `Path.get("${projectDir}", "..". "..". "TestData")` maybe.

I comment this here, but try to apply it to all paths.

Reply:
Makes sense.  I'll go through and try to fix these all up.